probert/tests/test_dasd.py

```diff
@@ -7,6 +7,36 @@
 from probert.tests.helpers import random_string
 
 
+# The tests parse canned dasdview output, and to be able to write
+# tests one needs to simply know what the correct parsed values
+# are. These helper functions help make the this dependence on outside
+# knowledge a little clearer:
+
+
+def expected_dasdd_probe_data(*, devname='/dev/dasdd', device_id):
+    return {
+        'blocksize': 4096,
+        'cylinders': 30051,
+        'device_id': device_id,
+        'disk_layout': 'cdl',
+        'name': devname,
+        'tracks_per_cylinder': 15,
+        'type': 'ECKD',
+        }
+
+
+def expected_dasde_probe_data(*, device_id):
```

Comment:
I do find dasdd and dasde to be hard to read embedded in that function name; almost like a typo.

```
expected_probe_data = {
   'dasdd': {...},
   'dasde': {...},
}

self.assertEqual(expected_probe_data['dasdd'], ...)

```

Reply:
Yeah that makes sense.